Broken docs-rs build when depending on flume

When using flume as a dependency in a project the build of the docs is broken because of an outdated config attribute in the dependencies.
Below an example of the error message:
```
Checking getrandom v0.2.16
error[E0557]: feature has been removed
Error:    --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/getrandom-0.2.16/src/lib.rs:211:29
    |
211 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
    |                             ^^^^^^^^^^^^ feature has been removed
    |
    = note: removed in 1.92.0; see <https://github.com/rust-lang/rust/pull/138907> for more information
    = note: merged into `doc_cfg`

For more information about this error, try `rustc --explain E0557`.
error: could not compile `getrandom` (lib) due to 1 previous error
```
Can this dependency be updated or replaced?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,8 +29,8 @@ default = ["async", "select", "eventual-fairness"]
 
 [dependencies]
 spin1 = { package = "spin", version = "0.9.8", features = ["mutex"] }
-futures-sink = { version = "0.3", default_features = false, optional = true }
-futures-core = { version = "0.3", default_features = false, optional = true }
+futures-sink = { version = "0.3", default-features = false, optional = true }
+futures-core = { version = "0.3", default-features = false, optional = true }
 fastrand = { version = "2.3", features = ["std", "js"], optional = true }
 
 [dev-dependencies]

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -838,7 +838,7 @@ impl<T> Drop for Sender<T> {
 /// all normal [`Sender`]s are dropped, the channel is closed, even if there is still a
 /// `WeakSender`.
 ///
-/// To send messages, a `WeakSender` must first be upgraded to a `Sender` using the [`upgrade`]
+/// To send messages, a `WeakSender` must first be upgraded to a `Sender` using the [`WeakSender::upgrade`]
 /// method.
 pub struct WeakSender<T> {
     shared: Weak<Shared<T>>,

